src/components/referrals/ImportReferralsComponent.vue

```diff
@@ -0,0 +1,180 @@
+                    id="uploadTest"
+                    type="file"
+                    @change="handleChange" />
+                <error-info :message="testFileError" />
+            </div>
+
+            <div class="actionsAndInfo">
+                <button-component
+                    :disabled="disableAddTest"
+                    wide
+                    >Zapisz skierowanie</button-component
+                >
+                <bouncing-balls-component :visible="isLoading" />
+                <error-info :message="error" />
+                <span
+                    v-show="successFlag"
+                    class="dataSend"
+                    >Skierowanie zostało przesłane pomyślnie</span
+                >
+            </div>
+        </form>
+    </div>
+</template>
+
+<script setup>
+    import ButtonComponent from '../ButtonComponent.vue';
+    import BouncingBallsComponent from '../BouncingBallsComponent.vue';
+    import ErrorInfo from '../ErrorInfo.vue';
+    import { computed, ref } from 'vue';
+    import ALLOWED_TYPES from '../../utils/allowedTypes.js';
+    import useStorage from '../../composables/getImage.js';
+    import useUserStore from '../../stores/userStore.js';
+    import { timestamp } from '../../firebase/config.js';
+    import useCollection from '../../composables/useCollections.js';
+    const { isLoading, addDoc, error } = useCollection('listOfImportReferrals');
+    const { url, uploadImage } = useStorage();
+    const nameAndSurnameDoctor = ref(null);
+    const researchDate = ref(null);
+    const successFlag = ref(false);
+    const testFile = ref(null);
+    const testFileError = ref(null);
+    const userStore = useUserStore();
+    const disableAddTest = computed(() => !testFile.value);
+    const dateFormat = computed(
+        () =>
+            researchDate.value &&
+            researchDate.value.split('-').reverse().join('-')
+    );
+    const handleChange = (event) => {
+        const selectedFile = event.target.files[0];
+        if (selectedFile && ALLOWED_TYPES.includes(selectedFile.type)) {
+            testFile.value = selectedFile;
+            testFileError.value = null;
+        } else {
+            testFile.value = null;
+            testFileError.value =
+                'Please select an image file (jpeg, jpg, png, webp or pdf).';
+        }
+    };
+    const addTest = async () => {
+        try {
+            await uploadImage(testFile.value);
+            console.log(`image uploaded`, url.value);
+            await addDoc({
+                doctorReferralsDataImport: nameAndSurnameDoctor.value,
+                userId: userStore.getUserId,
+                dateImport: dateFormat.value,
+                testUrl: url.value,
+                createdAt: timestamp(),
+            });
+            if (error.value) throw new Error();
+            isLoading.value = false;
+            successFlag.value = true;
+        } catch (err) {
+            console.log(err);
```

Comment:
why you display 3 console logs with erros

Reply:
If you import error from usecollection, you should display this error on display, and error handling in console